lib/pure/asyncftpclient.nim

```diff
@@ -253,15 +253,18 @@ proc retrText*(ftp: AsyncFtpClient, file: string): Future[string] {.async.} =
 
   result = await ftp.getLines()
 
-proc getFile(ftp: AsyncFtpClient, file: File, total: BiggestInt,
+proc getFile(ftp: AsyncFtpClient, dest: string, total: BiggestInt,
              onProgressChanged: ProgressChangedProc) {.async.} =
   assert ftp.dsockConnected
+  var file = open(dest, mode = fmWrite)
   var progress = 0
   var progressInSecond = 0
   var countdownFut = sleepAsync(1000)
+  const maxSpeed = 1024*1024*2
+  let BufferSize = if total > 0 and total < maxSpeed: total.int else: maxSpeed
```

Comment:
Haven't total size when not rfc959.
And maybe has 10G net speed at futrue, maybe don't want alloc 10G space at once haven't limit.
Of cause, It's not best way, I'm a nim newbie, They are temp way in my project.